src/xenia/kernel/xam/xam_module_export_groups.inc

```diff
@@ -27,3 +27,4 @@ XE_MODULE_EXPORT_GROUP(xam, UI)
 XE_MODULE_EXPORT_GROUP(xam, User)
 XE_MODULE_EXPORT_GROUP(xam, Video)
 XE_MODULE_EXPORT_GROUP(xam, Voice)
+XE_MODULE_EXPORT_GROUP(xam, Party)
```

Comment:
Did you mean Warty, Xarty, Yarty or Zarty?

Reply:
Done 🦊